agent/config/builder.go

```diff
@@ -787,124 +787,122 @@ func (b *Builder) Build() (rt RuntimeConfig, err error) {
-		SessionTTLMin:                           b.durationVal("session_ttl_min", c.SessionTTLMin),
-		SkipLeaveOnInt:                          skipLeaveOnInt,
-		StartJoinAddrsLAN:                       b.expandAllOptionalAddrs("start_join", c.StartJoinAddrsLAN),
-		StartJoinAddrsWAN:                       b.expandAllOptionalAddrs("start_join_wan", c.StartJoinAddrsWAN),
-		SyslogFacility:                          b.stringVal(c.SyslogFacility),
-		TLSCipherSuites:                         b.tlsCipherSuites("tls_cipher_suites", c.TLSCipherSuites),
-		TLSMinVersion:                           b.stringVal(c.TLSMinVersion),
-		TLSPreferServerCipherSuites:             b.boolVal(c.TLSPreferServerCipherSuites),
-		TaggedAddresses:                         c.TaggedAddresses,
-		TranslateWANAddrs:                       b.boolVal(c.TranslateWANAddrs),
-		UIDir:                                   b.stringVal(c.UIDir),
-		UIContentPath:                           UIPathBuilder(b.stringVal(b.Flags.Config.UIContentPath)),
-		UnixSocketGroup:                         b.stringVal(c.UnixSocket.Group),
-		UnixSocketMode:                          b.stringVal(c.UnixSocket.Mode),
-		UnixSocketUser:                          b.stringVal(c.UnixSocket.User),
-		VerifyIncoming:                          b.boolVal(c.VerifyIncoming),
-		VerifyIncomingHTTPS:                     b.boolVal(c.VerifyIncomingHTTPS),
-		VerifyIncomingRPC:                       b.boolVal(c.VerifyIncomingRPC),
-		VerifyOutgoing:                          verifyOutgoing,
-		VerifyServerHostname:                    verifyServerName,
-		Watches:                                 c.Watches,
+		AdvertiseAddrLAN:                 advertiseAddrLAN,
+		AdvertiseAddrWAN:                 advertiseAddrWAN,
+		BindAddr:                         bindAddr,
+		Bootstrap:                        b.boolVal(c.Bootstrap),
+		BootstrapExpect:                  b.intVal(c.BootstrapExpect),
+		CAFile:                           b.stringVal(c.CAFile),
+		CAPath:                           b.stringVal(c.CAPath),
+		CertFile:                         b.stringVal(c.CertFile),
+		CheckUpdateInterval:              b.durationVal("check_update_interval", c.CheckUpdateInterval),
+		CheckOutputMaxSize:               b.intValWithDefault(c.CheckOutputMaxSize, 4096),
+		Checks:                           checks,
+		ClientAddrs:                      clientAddrs,
+		ConfigEntryBootstrap:             configEntries,
+		AutoEncryptTLS:                   autoEncryptTLS,
+		AutoEncryptAllowTLS:              autoEncryptAllowTLS,
+		ConnectEnabled:                   connectEnabled,
+		ConnectCAProvider:                connectCAProvider,
+		ConnectCAConfig:                  connectCAConfig,
+		ConnectProxyBindMinPort:          proxyMinPort,
+		ConnectProxyBindMaxPort:          proxyMaxPort,
+		ConnectSidecarMinPort:            sidecarMinPort,
+		ConnectSidecarMaxPort:            sidecarMaxPort,
+		ConnectProxyDefaultExecMode:      proxyDefaultExecMode,
```

Comment:
I _think_ these 4 `ConnectProxy...` config options can also go.

Reply:
There's probably some more threads to pull when you remove these (like following their current values up and nuking those).